pkg/trace/operation.go

```diff
@@ -237,6 +237,14 @@ func NewOperation(ctx context.Context, format string, args ...interface{}) Opera
 	return o
 }
 
+func NewOperationFromID(ctx context.Context, ID string, format string, args ...interface{}) Operation {
+	o := newOperation(ctx, ID, 3, fmt.Sprintf(format, args...))
+
+	frame := o.t[0]
+	o.Debugf("[NewOperation] %s [%s:%d]", o.header(), frame.funcName, frame.lineNo)
```

Comment:
I think the use of `o.header()` may cause stuttering in the logs with the opid appearing twice.
I think this already occurs in one path iirc.